### DISCUSSION TITLE
Allow passing query client options to useQueryProvider

### DISCUSSION THREAD

**Discussion by henribru** [DamianOsipiuk/vue-query#37] [Ideas]
It would be convenient if `QueryClientConfig` could be passed to `useQueryProvider`, which would pass that on to the `QueryClient` constructor. Currently if you want to customize the query client you need to handle providing it yourself, I think?

I'd be happy to do a PR with this change if the idea is sound

**Comment by DamianOsipiuk** [comment]
Hi,
It's definitely a good idea. 💡 
Also in line with the main package.

PR is welcome 👍
